ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/dot1x.schema.yml

```diff
@@ -13,3 +13,23 @@ keys:
         type: bool
       dynamic_authorization:
         type: bool
+      mac_based_authentication:
```

Comment:
you can add the ranges from documentation in the schema as well using `min` and `max` (cf https://avd.sh/en/stable/docs/input-variable-validation-BETA.html#schema-options-for-type-int-integer)

Given that you have added them in the README, it is worth adding them in the schema